main.lua

```diff
@@ -230,6 +232,10 @@ function DtDisplay:showDateTimeWidget()
     UIManager:show(DisplayWidget:new {})
 end
```

Comment:
Unrelated to this PR but I don't think this actually gets used? I'll clean up in a followup.